samsungtvws/art.py

```diff
@@ -221,63 +268,78 @@ def upload(self, file, matte="shadowbox_polar", file_type="png", date=None):
         if date is None:
             date = datetime.now().strftime("%Y:%m:%d %H:%M:%S")
 
-        response = self._send_art_request(
-            {
-                "request": "send_image",
-                "file_type": file_type,
-                "conn_info": {
-                    "d2d_mode": "socket",
-                    "connection_id": random.randrange(4 * 1024 * 1024 * 1024),
-                    "id": self.art_uuid,
+        try:
+            response = self._send_art_request(
```

Comment:
I think here it would be better to split it completely into two methods:
```python
    try:
        self._upload_v1(file, matte, file_type, date)
    except exceptions.ResponseError:
        self._upload_v2(file, matte, file_type, date)
```

Reply:
won't have access to the TV 'till next month, will check later.
Thanks for the review!